Gruntfile.js

```diff
@@ -0,0 +1,59 @@
+module.exports = function (grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+        sass: {
+            options: {
+                sourceMap: true
+            },
+            dist: {
+                files: {
+                    'main.css': 'main.sass'
+                }
+            }
+        },
+
+        imagemin: {
+            dynamic: {
+                files: [{
+                    expand: true,
+                    cwd: 'images/',
+                    src: ['**/*.{png,jpg,gif}'],
+                    dest: 'images/build/'
```

Comment:
add few images to directory images/ and run grunt - test your configuration :) something is wrong here